src/compiler/scan.ts

```diff
@@ -57,7 +57,7 @@ export class Scan {
   private fullPath: string
   private withParameters?: string[]
   private requireConfig: boolean
-  private configSchema?: z.ZodType
+  private configSchema?: z.ZodTypeAny
```

Comment:
This is the only change in this PR. The rest is downgrading this package to zod v3